src/WorkOS.net/Services/MFA/_interfaces/ChallengeFactorOptions.cs

```diff
@@ -0,0 +1,13 @@
+ namespace WorkOS
+{
+    using Newtonsoft.Json;
+
+    public class ChallengeFactorOptions : BaseOptions
+    {
+        /// <summary>
+        /// Authentication Factor ID.
+        /// </summary>
+        [JsonProperty("authentication_factor_id")]
+        public string Type { get; set; }
```

Comment:
Should this be `AuthenticationFactorId`?

```suggestion
        public string AuthenticationFactorId { get; set; }
```

Reply:
Or, perhaps shortened:

```suggestion
        public string FactorId { get; set; }
```